libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustMatrixRoom.kt

```diff
@@ -268,9 +268,10 @@ class RustMatrixRoom(
             innerRoom.timelineWithConfiguration(
                 configuration = TimelineConfiguration(
                     focus = focus,
-                    allowedMessageTypes = allowedMessageTypes,
+                    filter = filter,
                     internalIdPrefix = internalIdPrefix,
                     dateDividerMode = dateDividerMode,
+                    trackReadReceipts = false,
```

Comment:
According to [this](https://github.com/matrix-org/matrix-rust-sdk/pull/4735#discussion_r1975610992), this should always be false for now